ocp-templates/README.md

```diff
@@ -16,6 +18,9 @@ The prod templates are set up to use the images at [quay.io/organization/ibutsu]
 ``latest``. These images are built and tagged as ``latest`` on every release of Ibutsu. These are meant to deploy the production instance of Ibutsu. The templates include some extra things like database
 backups and vacuums.
 
+## Prod
+The MPP templates are meant to be used on an OCPv4 Openshift cluster with multi-tennant support. 
```

Comment:
I think better not to use specific cluster `multi-tenant` arch. It will useful to anyone where they want to go with `keycloak` signing.

Reply:
agreed